runtime/src/bank.rs

```diff
@@ -124,7 +124,7 @@ pub struct Bank {
     parent: RwLock<Option<Arc<Bank>>>,
 
     /// The set of parents including this bank
-    ancestors: HashMap<u64, usize>,
+    pub ancestors: HashMap<u64, usize>,
```

Comment:
how come not a pub method instead?

Reply:
this isn't Java, we don't need get/set methods for everything

Reply:
looks like the only customer only wants the parents' slots...